server/accesslog.go

```diff
@@ -126,10 +126,17 @@ func (a *AccessLogServer) handleHTTPLogs(msg *als.StreamAccessLogsMessage_HttpLo
 
 		var api string
 		var authContext *auth.Context
+		var grpcService string
+		var operation string
 
 		extAuthzMetadata := getMetadata(extAuthzFilterNamespace)
 		if extAuthzMetadata != nil {
-			api, authContext = a.handler.decodeAuthMetadata(extAuthzMetadata.GetFields())
+			metadata, err := a.handler.decodeAuthMetadata(extAuthzMetadata.GetFields())
+			if err != nil {
+				log.Debugf(err.Error())
+				continue
```

Comment:
Having the continue here prevent the `log.Debugf...` on line 149 of the changed version from running.  Im not sure if I should remove the continue or not.

Reply:
Hmm. Maybe we could just consolidate all these err checks and related logging to after this if/else block?

Reply:
I pushed a changed that consolidates the err checks and related logging to after this if/else block.